src/explorer/helpTree.ts

```diff
@@ -97,26 +97,28 @@ export default class HelpTree
 
       const feedback = new HelpLinkTreeItem(
         'Suggest a Feature',
-        'https://feedback.mongodb.com/forums/929236-mongodb-for-vs-code/',
+        LINKS.feedback,
         'feedback',
         'lightbulb'
       );
 
       const reportBug = new HelpLinkTreeItem(
         'Report a Bug',
-        'https://github.com/mongodb-js/vscode/issues',
+        LINKS.reportBug,
         'reportABug',
         'report'
       );
 
       const telemetryUserIdentity =
         this._telemetryService?.getTelemetryUserIdentity();
-      const ajsAid = telemetryUserIdentity
-        ? `&ajs_aid=${telemetryUserIdentity[0]}`
```

Comment:
Related ongoing slack thread: https://mongodb.slack.com/archives/CJ9105ZSA/p1685617300511319

Reply:
I think this one https://segment.com/docs/connections/sources/catalog/libraries/website/javascript/querystring/
Since we use `ajs_aid` it refers to `anonymousId`.